api/config/config_local.json

```diff
@@ -13,7 +13,8 @@
     "xAppIdValue": "local-AoU-RW",
     "vpcServicePerimeterName": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test",
     "timeoutInSeconds": 60,
-    "jupyterDockerImage": "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.6"
```

Comment:
Version 0.0.8 is the first to be published to Dockerhub
By default Welder is pulled from GCR, so we need to be explicit